en/SetupView/Joystick.md

```diff
@@ -30,12 +30,17 @@ To configure a joystick:
 
    ![Joystick setup - PlayStation](../../assets/setup/joystick_sony_playstation.jpg)
```

Comment:
Why is this file deleted in the list above?

Reply:
Do you mean the files list? It is not deleted. I think it is in red because the file size shrunk

Reply:
Yes I did. Good point. Thank you.